app/utils/markdown/index.ts

```diff
@@ -20,7 +20,7 @@ type LanguageObject = {
 
 // pattern to detect the existence of a Chinese, Japanese, or Korean character in a string
 // http://stackoverflow.com/questions/15033196/using-javascript-to-check-whether-a-string-contains-japanese-characters-includi
-const cjkPattern = /[\u3000-\u303f\u3040-\u309f\u30a0-\u30ff\uff00-\uff9f\u4e00-\u9faf\u3400-\u4dbf\uac00-\ud7a3]/;
+const cjkPattern = /[\u3000-\u303f\u3040-\u309f\u30a0-\u30ff\uff00-\uff9f\u4e00-\u9faf\u3400-\u4dbf\uac00-\ud7a3\u0e00-\u0e7f]/;
```

Comment:
I wonder if we should update the variable name to `cjktPattern`. 0/5

Reply:
I'd have to let experienced Mattermost developers handle that -- not sure if it'd break something somewhere else.

Maybe better something like extraPattern? In case some more languages turn up later..

Reply:
Changing the comment works for me. We can always change it again in the future if needed